monai/handlers/checkpoint_loader.py

```diff
@@ -53,9 +54,12 @@ def __init__(
         load_dict: Dict,
         name: Optional[str] = None,
         map_location: Optional[Dict] = None,
+        find_latest: bool = False,
     ) -> None:
         assert load_path is not None, "must provide clear path to load checkpoint."
         self.load_path = load_path
+        assert (find_latest and self.load_path[-3:] != '.pt') or \
+               self.load_path[-3:] == '.pt' and not find_latest, "either provide an exact path, or set find latest to true"
```

Comment:
This could be simplified to an XOR (one or the other, but not both). In python, this is the `^` operator.